core/api/src/app/payments/send-intraledger.ts

```diff
@@ -245,13 +270,19 @@ const executePaymentViaIntraledger = async <
     walletId: recipientWalletDescriptor.id,
     userId: recipientAccount.kratosUserId,
     level: recipientAccount.level,
+    status: recipientAccount.status,
+    username: recipientAccount.username,
```

Comment:
are we using this? also this does not make sense because recipient will not have username